### PR TITLE
AP-4646: SCA check who your client is

### PR DESCRIPTION
**Pull request by agoldstone93**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4646)

- Add check who your client is CYA page
- Update forms for Qs 1-3 so they behave as expected
- Add check_who_your_client_is step and step spec

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/shared/check_answers/_relationship_to_child.html.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        row.with_key(text: t(".biological_parent"), classes: "govuk-!-width-one-half")
+        row.with_value(text: yes_no(@relationship_to_child.eql?("biological")))
+        row.with_action(text: "Change", href: providers_merits_task_list_is_client_biological_parent_path(@proceeding), visually_hidden_text: t(".biological_parent"))
+      end
+
+      unless @relationship_to_child.eql?("biological")
+        summary_list.with_row do |row|
+          row.with_key(text: t(".parental_responsibilities"))
+          row.with_value(text: @relationship_to_child.nil? ? t("generic.no") : t(".#{@relationship_to_child}"))
+          row.with_action(text: "Change", href: providers_merits_task_list_does_client_have_parental_responsibility_path(@proceeding), visually_hidden_text: t(".parental_responsibilities"))
```
Comment (kmahern) [inline app/views/shared/check_answers/_relationship_to_child.html.erb:12]: Should the text be a translation ie t("generic.change")

File: spec/requests/providers/proceeding_merits_task/is_client_child_subject_controller_spec.rb [hunk 2]
```diff
@@ -73,10 +73,9 @@ module ProceedingMeritsTask
             expect(legal_aid_application.legal_framework_merits_task_list).to have_not_started_task(:PB003, :client_relationship_to_proceeding)
           end
 
-          # TODO: change the following to redirect to question 4 when it is added
           it "redirects to merits task list page" do
```
Comment (kmahern) [inline spec/requests/providers/proceeding_merits_task/is_client_child_subject_controller_spec.rb:76]: should it be it "redirects to check who client is" page?

File: app/controllers/providers/proceeding_merits_task/does_client_have_parental_responsibilities_controller.rb [hunk 3]
```diff
@@ -7,7 +7,10 @@ def show
 
       def update
         @form = Providers::ProceedingMeritsTask::ParentalResponsibilitiesForm.new(form_params)
-        return redirect_to providers_merits_task_list_is_client_child_subject_path(merits_task_list_id) if @form.relationship_to_child.eql?("false")
+        if @form.relationship_to_child.eql?("false")
```
Comment (kmahern) [inline app/controllers/providers/proceeding_merits_task/does_client_have_parental_responsibilities_controller.rb:10]: Is there a reason why this controller doesn't have the && !draft_selected and the other 2 do? Do we need to add it?

Comment (agoldstone93) [reply app/controllers/providers/proceeding_merits_task/does_client_have_parental_responsibilities_controller.rb]: Added

File: spec/requests/providers/proceeding_merits_task/is_client_child_subject_controller_spec.rb (hunk 2 above)
Comment (agoldstone93) [reply spec/requests/providers/proceeding_merits_task/is_client_child_subject_controller_spec.rb]: Yep, I've made the change

File: app/views/shared/check_answers/_relationship_to_child.html.erb (hunk 1 above)
Comment (agoldstone93) [reply app/views/shared/check_answers/_relationship_to_child.html.erb]: Yep, good spot

**Review by kmahern — APPROVED** [review]
lgtm 👍

**Review by skdejong — APPROVED** [review]

**Pull request merged**